pages/requirements/curriculum-guide.js

```diff
@@ -21,17 +27,17 @@ const CurriculumGuide = ({ curriculumGuides }) => {
         <title>چارت درسی | انجمن کامپیوتر صنعتی ارومیه</title>
       </Head>
       <h2>چارت</h2>
-      {curriculumGuides.map((curriculumGuide, curriculumGuideIndex) => (
```

Comment:
The reason We map through `curriculumGuides` is that there will be several of them in future; But currently there is just one.  
Also notice in line 14 `curriculumGuides[0]`. As I mentioned may be several `curriculumGuide` inside `curriculumGuides` array.